src/components/CharacterLimitMainText/CharacterLimitMainText.tsx

```diff
@@ -0,0 +1,22 @@
+import CharacterLimit from './components/CharacterLimit';
+
+import scss from './CharacterLimitMainText.module.scss';
+
+import { ICharacterLimitMainTextProps } from './CharacterLimitMainText.types';
+
+function CharacterLimitMainText(props: ICharacterLimitMainTextProps) {
+  return (
+    <div className={scss.compost}>
+      {props.module.map((module) => (
+        <CharacterLimit
+          maxLength={module.maxLength}
+          value={module.value}
+          id={module.id}
+          svg={module.svg}
+        />
+      ))}
```

Comment:
Se esse module for um array vazio o que vai acontecer?

Reply:
renderiza um container vazio kk